src/mame/midway/williams.cpp

```diff
@@ -3130,7 +3205,17 @@ Although not currently dumped, "true" rev.1 Sinistar ROMs are believed to be num
 There is known to be a "perfect" version of Sinistar, that being the original version presented to Williams by the
   dev team. The dev team thought this version had the best game play while Williams decided it was too easy (IE: it
   could be played too long on one quarter)
-
+  
+Sinistar's cockpit cabinet features two sound boards, one for the front speakers and another for the rear.  The rear
+  sound board uses a different ROM, Video Sound ROM 10.  This is nearly identicial to Video Sound ROM 9, except for 
+  a routine that adds a slight delay to some of the sound effects which produces the stereo separation effect.
+It also disables the extra ship and bounce sound effects.  As seen in the drawing set for Sinistar, there are no
+  speech ROMs connected to the rear board, so only the front speakers play speech.
+  
+If you disconnect the speech ROMs from the upright sound board, Video Sound ROM 9 will play two replacement sound
+  effects for the Sinistar's missing audio.  Any line of dialogue will be replaced by a generic alarm noise,
+  while the Sinistar roar is replaced by a loud square wave synth noise that attempts to emulate the "sini-scream".
+  Video Sound Rom 10 disables this functionality so that it doesn't play any error sounds during speech calls.
```

Comment:
There’s stray whitespace on lines here.  Please use the `srcclean` tool to normalise spacing in the file.